src/frontend/Komponenter/Journalføring/BehandlingKlageInnold.tsx

```diff
@@ -0,0 +1,144 @@
+        <>
+            <Systemtittel>Behandling</Systemtittel>
+            <table className="tabell">
+                <thead>
+                    <tr>
+                        <th></th>
+                        <th>Status</th>
+                        <th>Sist endret</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <DataViewer response={{ behandlinger }}>
+                        {({ behandlinger }) => (
+                            <>
+                                {valgtFagsak &&
+                                    behandlinger[valgtFagsak].map(
+                                        (behandlingsEl: KlageBehandling) => (
+                                            <tr key={behandlingsEl.id}>
+                                                <td>
+                                                    <Checkbox
+                                                        onChange={håndterCheck(behandlingsEl.id)}
+                                                        checked={
+                                                            behandlingsEl.id ===
+                                                            behandling?.behandlingId
+                                                        }
+                                                        label={behandlingsEl.status}
+                                                    />
+                                                </td>
+                                                <td>{behandlingsEl.status}</td>
+                                                <td>
+                                                    {formaterNullableIsoDato(
+                                                        behandlingsEl.vedtaksdato
+                                                    )}
+                                                </td>
+                                            </tr>
+                                        )
+                                    )}
+                                {nyBehandling && (
+                                    <StyledNyBehandlingRad>
+                                        <td>
+                                            <Checkbox
+                                                onChange={håndterCheck('ny')}
+                                                checked={harValgtNyBehandling}
+                                                label={'ny'}
+                                            />
+                                        </td>
+                                        <td>NY</td>
+                                        <td>–</td>
+                                    </StyledNyBehandlingRad>
+                                )}
+                            </>
+                        )}
+                    </DataViewer>
+                </tbody>
+            </table>
+            {!behandling?.behandlingId && (
+                <Flatknapp onClick={lagNyBehandlingRad}>
```

Comment:
Kunne vi brukt Nav-ds sin button her? Husk å legg til `type='button'`

Reply:
✅  Fikset det for vanlig behandling sin og